### ISSUE TITLE
can not remote ubuntu 24.04 which uses gnome-remote-desktop

### ISSUE THREAD

**Issue by Minsdata** [jwetzell/docker-guacamole#58]
**Describe The Bug:**
can not remote ubuntu 24.04 which uses gnome-remote-desktop
its ok for ubuntu 22 which uses xrdp, but i cant remote the linux pc (ubuntu 24.04)

**Comment by jwetzell** [comment]
I believe this is related to some things that are supposed to be addressed in guacamole 1.6.0 if you have logs from both sides that might help verify that.

**Comment by Minsdata** [comment]
here is the log when i use guacamole to remote ubuntu

Dec 23 11:26:36 ubuntu xrdp[4120]: [INFO ] lib_mod_connect: connecting via UNIX socket
Dec 23 11:26:37 ubuntu xrdp[4120]: [INFO ] lib_mod_log_peer: xrdp_pid=4120 connected to X11rdp_pid=4124 X11>
Dec 23 11:26:37 ubuntu xrdp[4120]: [INFO ] connected ok
Dec 23 11:26:41 ubuntu xrdp[4120]: [WARN ] xrdp_mm_chansrv_connect: connect failed trying again...
Dec 23 11:26:45 ubuntu xrdp[4120]: [WARN ] xrdp_mm_chansrv_connect: connect failed trying again...
Dec 23 11:26:49 ubuntu xrdp[4120]: [WARN ] xrdp_mm_chansrv_connect: connect failed trying again...
Dec 23 11:26:53 ubuntu xrdp[4120]: [WARN ] xrdp_mm_chansrv_connect: connect failed trying again...
Dec 23 11:26:53 ubuntu xrdp[4120]: [ERROR] xrdp_mm_chansrv_connect: error in trans_connect chan
Dec 23 11:26:53 ubuntu xrdp[4120]: [ERROR] SSL_shutdown: Failure in SSL library (protocol error?

**Comment by jwetzell** [comment]
Yep, this looks like the same issue I faced. Going to have to wait until Guacamole 1.6.0.

**Comment by jwetzell** [comment]
1.6.0 seems to have solved this, tested before and after with an Ubuntu VM and it seems to be working now.
